### ISSUE TITLE
Show as a list instead of dropdown

### ISSUE THREAD

**Issue by hy-net** [snapappointments/bootstrap-select#2866]
Option to show as a list instead of dropdown. It will be very convenient from time to time.

**Comment by hrvoj3e** [comment]
Yes. This would be great. Scrollable list instead of dropdown just like how a vanilla `<select multiple size="6">` works in browsers.

**Comment by santoshbmath** [comment]
I agree, it would be great. "size" doesnt work similar to select in bootstrap. It always displays dropdown.
